<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QDialog" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>126</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>288</width>
    <height>126</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>288</width>
    <height>126</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Login :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Mot de passe :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditLogin">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(212, 212, 212);</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMDP">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(212, 212, 212);</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelError">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string>Login ou mot de passe incorrect</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonConnect">
         <property name="text">
          <string>Connection</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAnnuler">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Annuler</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
